app/views/global_issue_templates/new.html.erb

```diff
@@ -1,18 +1,17 @@
 <div class="contextual">
   <%= link_to(l(:label_list_templates),
-              {:controller => :global_issue_templates, :action => 'index'
-              }, :class => 'template') %>
+              { controller: 'global_issue_templates', action: 'index' }, class: 'template') %>
 </div>
 <h2 class="template"><%=h "#{l(:issue_templates)} / #{l(:button_add)}" %></h2>
 
 <div class="box">
   <%= error_messages_for 'global_issue_template' %>
   <%= labelled_form_for :global_issue_template,  @global_issue_template,
-                        :url => { :controller => 'global_issue_templates', :action => 'new' },
-                        :html => { :id => 'global_issue_template-form',
-                                   :class => nil, :multipart => false } do |f| %>
+                        url: { controller: 'global_issue_templates', action: 'new' },
+                        html: { id: 'global_issue_template-form',
+                                   class: nil, multipart: false } do |f| %>
 
-      <%= render :partial => 'form', :locals => { :f => f } %>
+      <%= render partial: 'form', locals: { f: f } %>

```

Comment:
__[rails_best_practices]__
* simplify render in views [:ref](http://rails-bestpractices.com/posts/2010/12/04/simplify-render-in-views/)